src/main/java/selenium/pages/BurgerMenu.java

```diff
@@ -0,0 +1,25 @@
+package selenium.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static java.time.Duration.ofSeconds;
+
+public class BurgerMenu extends BasePage {
+    @FindBy(id = "react-burger-menu-btn")
+    public WebElement burgerMenu;
+
+    @FindBy(id = "logout_sidebar_link")
+    public WebElement logout;
```

Comment:
Please provide more understandable name for a fields according to Java code convention. logoutLink is a correct name for this field

Reply:
Done